src/App.tsx

```diff
@@ -27,22 +27,20 @@ const pageContainer = classnames(
   space('space-y-6', 'sm:space-y-0'),
   display('flex'),
   flexDirection('flex-col'),
-  minHeight('min-h-screen'),
-  justifyContent('justify-between')
+  minHeight('min-h-screen')
 )
+const mainContent = classnames(space('space-y-4'), margin('sm:!my-auto'))
```

Comment:
I know that using `important` is wrong, but it's the only way how to overwrite `sm:space-y-0` properties and make the space between header/main content/footer responsive